scripts/unoptimized-deployer-meta

```diff
@@ -1,6 +1,6 @@
 {
-  "factory_hash": "0x8e3dc1fb25976a8279ce40a4b2c3d182158365da7c579cde433f5d8164691bf2",
-  "constant_product_hash": "0x6e3b13293eee4f98834bcde1f85c460a11a9323ce73efc080957044db4044c21",
-  "stable_hash": "0x3ca08204c301d01a2a143e86060aa7e8bcd991205dda2369d3b8899f9844b0c2",
+  "factory_hash": "0x542fe795ae57ecd8fb2233c98ae521ab2973ea1e286e679be44f573b1ec571a4",
+  "stable_hash": "0x4ac793675e33c3fe47b307e7ef3f14f35f5855c5f86244356b39f011be1de722",
+  "constant_product_hash": "0x0d0f6c6e5b513a7dfc8380dc45d750502c8a69b9122c62bcb967dff7a749b83f",
```

Comment:
now deterministic: https://github.com/foundry-rs/foundry/issues/4592

Pushing the latest version